Products/Poi/content/tracker.py

```diff
@@ -389,7 +389,7 @@ def getTagsInUse(self):
         issues = self.getIssues()
         tags = {}
         for i in issues:
-            for s in i.subject:
+            for s in i.Subject:
```

Comment:
Oh, because that’s the brain and not the actual object. Makes sense. (I feel like rewriting that whole function.)